### DISCUSSION TITLE
Add a target within a remote OpenVPN network?

### DISCUSSION THREAD

**Discussion by senpro-ingwersenk** [warp-tech/warpgate#1113] [Q&A]
Hello there!

Quick and short question: Assume that I have a target I want to access, but it is in a remote network, which I can only access after connecting via OpenVPN.

How would I add such a target?

Thanks and kind regards!

**Comment by Eugeny** [comment]
It generally doesn't matter where the target is as long as Warpgate itself can reach it. You'd just need an always-on openvpn connection on the machine where Warpgate is running
